scim/bulkRecords.bal

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# + failOnErrors - No. of errors to fail the bulk request
+# + schemas - URIs of the SCIM schemas used in the request  
+# + Operations - List of operations
+public type Bulk record {
+    int failOnErrors?;
+    string[] schemas;
+    Operation[] Operations;
+};
+
+# + method - method of the operation  
+# + path - path of the operation 
+# + bulkId - bulkId of the operation 
+# + data - data of the operation
+public type Operation record {
+    string method;
+    string path;
+    string bulkId?;
+    SCIMUser|BulkUserUpdate|BulkUserReplace|SCIMGroup|BulkGroupUpdate|BulkGroupReplace data?;
+};
+
+# + schemas - URIs of the SCIM schemas used
+# + username - username of the user 
+# + name - name of the user
+public type BulkUserReplace record {
+    string[] schemas;
+    string username;
+    Name name;
+};
+
+# + op - operation to be performed  
+# + path - path of the existing users
+# + value - value to be updated
+public type BulkUserUpdate record {
+    string op;
+    string path;
+    string value;
```

Comment:
The value should be even int, boolean, array, JSON ... 
any data type defined in https://www.rfc-editor.org/rfc/rfc7643.html#section-2.3
How that is handled y defining attribute value to string?